### DISCUSSION TITLE
Welcome to Temporal-Abstract-Spec Discussions!

### DISCUSSION THREAD

**Discussion by gbuehler** [opengeospatial/Temporal-Abstract-Spec#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chris-little** [comment]
Hello Everyone, 
I am @chris-little , Chair of the OGC Temporal Domain Working Group. My Co-chairs are Chuck Heazel and Ron Tse.
Many years ago, with the previous co-chairs, we set out to write an OGC Best Practice document, but life got in the way.

Meanwhile, various interesting things happened:
- W3C Spatial Data on the Web Working Group updated the W3C Time Ontology in OWL recommendation, and added a couple of extra Notes to improve usability;
- ISO TC211 committee updated the ISO19111:2019 Referencing by Coordinates standard to include temporal coordinates, thus making the original ISO19108:2002- Geographic Information - Temporal Schema largely redundant;
- ISO TC134 committee updated the ISO8601 standard with a Part 2 describng more complex notations for intervals and temporal patterns;
- ISO TC134 published two technical notes proposing standard codes for registering timescales and calendars;
- ISO TC211 and TC134 established a joint AdHoc Group on Time, AHG9, with Ron as the Chair.

After some recent meetings of the OGC Temporal Domain Working Group, we agreed to try to produce an Abstract Conceptual Model for Time, as requested by the OGC Architecture Board, as well as the Best Practice document.

The key feature of an Abstract Conceptual Model is that the only important things are the components and their relations to each other, and the important, or even only, attributes of the components are Identifiers.

Please join in.
